release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2832,6 +2825,107 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously on Red{nbsp}Hat OpenShift Service on AWS (ROSA) with hosted control planes (HCP), a cluster that used mirroring release images migth result in existing node pools to use the hosted cluster OS version instead of the NodePool version. With this release, a fix ensures that node pools use their own versions. (link:https://issues.redhat.com/browse/OCPBUGS-41552[*OCPBUGS-41552*])
+
+* Previously, using DNF to install additional packages on your customized {op-system-first} builds caused builds to fail because packages could not be located. With this release, the Subscription Manager adds the correct packages to {op-system} so that this issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-41376[*OCPBUGS-41376*])
+
+* Previosuly, bonds that were configured in `active-backup` mode would have EFI System Partition (ESP) offload active even if underlying links did not support ESP offload. This caused IPsec associations to fail. With this release, ESP offload is disabled for bonds so that IPsec associations pass. (link:https://issues.redhat.com/browse/OCPBUGS-41255[*OCPBUGS-41255*])
+
+* Previosuly, resources for new user account were not removed when the account was deleted. This caused unncessary information in config maps, roles and role-bindings. With this release, an `ownerRef` tag is added to these resources, so that when you delete a user account the resources are also deleted from all cluster resources. (link:https://issues.redhat.com/browse/OCPBUGS-39601[*OCPBUGS-39601*])
+
+* Previosuly, a coding issue caused ansible script on {op-system} user-provisioned installation infrastructure to fail. This occured when IPV6 was enabled for a three-node cluster. With this release, supprt exists for installating a three-node cluster that has Ipv6 enabled on {op-system}. (link:https://issues.redhat.com/browse/OCPBUGS-39409[*OCPBUGS-39409*])
+
+* Previously, the order of an Ansible playbook was modified to run before the `metadata.json` file was created, which caused issues with older versions of Ansible. With this release, the playbook is more tolerant of missing files and the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-39286[*OCPBUGS-39286*])
+
+* Previously, when the Node Tuning Operator (NTO) was configured to use `PerformanceProfiles`, NTO would create an `ocp-tuned-one-shot systemd` service. The systemd service would run prior to kubelet and blocked execution. The systemd service invokes Podman which uses an NTO image, but when the NTO image was not present Podman still tried to fetch the image and it would fail. With this release, support is added for cluster-wide proxy environment variables defined in `/etc/mco/proxy.env`. Now, Podman pulls NTO images in environments which need to use proxies for out-of-cluster connections. (link:https://issues.redhat.com/browse/OCPBUGS-39124[*OCPBUGS-39124*])
```

Comment:
```suggestion
* Previously, when the Node Tuning Operator (NTO) was configured to use `PerformanceProfiles`, NTO would create an `ocp-tuned-one-shot `systemd` service. The systemd service would run before Kubelet and blocked execution. The systemd` service invokes Podman which uses an NTO image, but when the NTO image was not present Podman still tried to fetch the image and it would fail. With this release, support is added for cluster-wide proxy environment variables defined in `/etc/mco/proxy.env`. Now, Podman pulls NTO images in environments which need to use proxies for out-of-cluster connections. (link:https://issues.redhat.com/browse/OCPBUGS-39124[*OCPBUGS-39124*])
```
See SSG" https://www.ibm.com/docs/en/ibm-style?topic=word-usage#p